### DISCUSSION TITLE
Student back benefits took too long

### DISCUSSION THREAD

**Discussion by Ahmad-HB** [community/community#143642] [GitHub Education] [answered]
### Hi everyone,

Hello everyone,

I’m excited to share that my application for the GitHub Education benefits was approved,  However, it seems that I haven’t yet received access to the benefits package, which includes the tools and features I hoped to introduce to support my learning.

<img width="307" alt="Screenshot 2024-11-05 at 3 15 25 PM" src="https://github.com/user-attachments/assets/984b03c4-8586-448d-905a-ed818f5fafab">

Could anyone from the community share insights or a similar experience? Additionally, if there are any next steps I need to complete, I’d appreciate any guidance to ensure my students can start benefiting from these tools as soon as possible. Thank you!

**Comment by server-craftsman** [comment]
Response #2

**Comment by davevad93** [comment, marked as answer]
Hi @Ahmad-HB ,

Don't worry, the waiting time may vary depending on different factors, such as the volume of requests; it usually takes a few days, but sometimes it can take a little longer, even a couple of weeks.

When the benefits become available, the status label will turn purple and change to:

- `Congrats! Your request for @Ahmad-HB was approved and your benefits are now available.`

For further reference you can look at these discussions: #132790 , #131665 , #132470 , #133531, #134279, #135849, #136887, #138721, #139268

And also the [Student Developer Pack Application FAQs](https://github.com/orgs/community/discussions/111352) :

- [How long after I submit my application will it be approved?](https://github.com/orgs/community/discussions/111352#how-long-after-i-submit-my-application-will-it-be-approved)
- [How long after I’ve been approved will I receive my academic benefits?](https://github.com/orgs/community/discussions/111352#how-long-after-ive-been-approved-will-i-receive-my-academic-benefits)
